### ISSUE TITLE
List of contributors?

### ISSUE THREAD

**Issue by schymane** [MassBank/MassBank-documentation#2]
It would be nice to have an easily visible list of contributors to MassBank in the documentation and displayed on the web if possible 
If there is one, I have not been able to navigate our many repos to find it easily ... 
We used to have a table displayed on massbank.jp, but this is no longer accessible through the new website. 
@meier-rene @sneumann @tsufz ideas?

**Comment by meier-rene** [comment]
Yes there is one: https://github.com/MassBank/MassBank-data/blob/main/List_of_Contributors_Prefixes_and_Projects.md . I will have a look for a nice place of a link to this list.

**Comment by schymane** [comment]
Thanks @meier-rene that would be great. Lux is not yet on it ... I am not sure if we already agreed on folder names with @tsufz 

Do you have anything on record your side, or should I update this list with my suggestions?

**Comment by meier-rene** [comment]
I cant remember any special issues related to this list and names. Feel free to use what ever is a appropriate in your eyes.

**Comment by schymane** [comment]
I am blind ... it's there already d'oh ... ;-) 
OK ... one step closer to validation ...

**Comment by tsufz** [comment]
@meier-rene, yes this was coming also in my mind. We need a place for outlinks anyway, maybe on the future about page (or old school link page... :-/ ).
